src/theme.js

```diff
@@ -151,6 +151,17 @@ export const duration = {
   slowest: 600
 }
 
+// animation easing curves
+const easeInOut = 'cubic-bezier(0.5, 0, 0.25, 1)'
+const easeOut = 'cubic-bezier(0, 0, 0.25, 1)'
+const easeIn = 'cubic-bezier(0.5, 0, 1, 1)'
+
+const easing = {
```

Comment:
I wonder if this should be called `timingFunctions` to stick with the CSS naming convention, e.g. `animation-timing-function: ${easeInOut}`

Reply:
Good call